auto_install/install.sh

```diff
@@ -745,9 +745,12 @@ setCustomDomain() {
 }
 
 confOpenVPN() {
-    # Generate a random, alphanumeric identifier of 16 characters for this server so that we can use verify-x509-name later that is unique for this server installation. Source: Earthgecko (https://gist.github.com/earthgecko/3089509)
+    # Grab the existing Hostname
+	HOST_NAME=$(hostname)
```

Comment:
@IcedComputer 
AFAIK the output of `hostname` without option is not consistent across distros and versions. You might want to switch to either `hostname -f` (FQDN) or `hostname -s` (assures short form only, without domain appending): https://linux.die.net/man/1/hostname

Reply:
added the -s to assure the short form of the host is used.